app/libs/utils.js

```diff
@@ -0,0 +1,15 @@
+export const timeToString = function (time) {
+    var totalSec = Math.floor(time);
+    var hours = parseInt( totalSec / 3600 ) % 24;
```

Comment:
`parseInt` is useless here (`totalSec` is a number, so `totalSec / <x>` where `<x>` is a number will always produce a number)